docs/docs/contributing/documentation/index.mdx

```diff
@@ -0,0 +1,7 @@
+# Contribute Documentation
```

Comment:
dont know if "Contribute" is needed, we're already in the Contribution section

Reply:
The other sections don't make sense when prefixed with `Contributing` - looks a bit funny